bazel/boringssl_static.patch

```diff
@@ -2,23 +2,33 @@ diff --git a/BUILD b/BUILD
 index d7c731bf6..315cdeca0 100644
 --- a/BUILD
 +++ b/BUILD
-@@ -88,6 +88,7 @@ boringssl_copts = select({
+@@ -65,6 +65,9 @@ boringssl_copts = select({
+     # This is needed on Linux systems (at least) to get rwlock in pthread.
+     "-D_XOPEN_SOURCE=700",
+ 
++    # Disable assembly code for testing cross-compilation
++    "-DOPENSSL_NO_ASM",
```

Comment:
It's hard to tell from this patch, is the setting architecture-specific or not? @mrostecki my version applied this setting to all Linux builds, if you haven't touched it will need to change.

Reply:
Good point. I didn't change anything in your boringssl-related commit, so it's not specifuc architecture. I will change that and enable this option only for arm.

Reply:
It would be good open a follow-up issue to resolve the underlying problem later on. I am pretty sure BoringSSL/OpenSSL are definitely meant to compile for AArch64 with assembly code, it's either Bazel compilation that is broken somehow, or perhaps just the particular commit.

Reply:
Good idea. For now I will just add `-DOPENSSL_NO_ASM` to the aarch64 toolchain with a comment.